airbyte-integrations/connectors/source-notion/unit_tests/test_streams.py

```diff
@@ -76,3 +77,61 @@ def test_backoff_time(patch_base_class):
     stream = NotionStream(config=MagicMock())
     expected_backoff_time = None
     assert stream.backoff_time(response_mock) == expected_backoff_time
+
+
+def test_users_request_params(patch_base_class):
+    stream = Users(config=MagicMock())
+
+    # No next_page_token. First pull
+    inputs = {"stream_slice": None, "stream_state": None, "next_page_token": None}
+    expected_params = {"page_size": 100}
+    assert stream.request_params(**inputs) == expected_params
+
+    # When getting pages after the first pull.
+    inputs = {"stream_slice": None, "stream_state": None, "next_page_token": {"next_cursor": "123"}}
+    expected_params = {"start_cursor": "123", "page_size": 100}
```

Comment:
Instead of using a dummy **_start_cursor_** value maybe use a self documenting value that will be provided from **_Notion_** such as  **_"next_cursor": "fe2cc560-036c-44cd-90e8-294d5a74cebc"_**

Reply:
You're right. Updated in [fda250e](https://github.com/airbytehq/airbyte/pull/11452/commits/fda250ecf078624eff22eb745c3eaa7337c947ce).